.travis.yml

```diff
@@ -30,6 +30,8 @@ script:
   - conda install --yes --quiet nose nose-timer
   # Test the package
   - cd devtools && nosetests $PACKAGENAME --nocapture --verbosity=2 --with-timer -a '!slow' && cd ..
+  # Run quickmodel
+  - pushd . && cd examples/direct-fluorescence-assay && env PYTHONPATH="./" quickmodel --inputs 'inputs_p38_singlet' && popd
```

Comment:
It's great that `quickmodel` has been added to travis. However, `quickmodel` runs the default number of MCMC steps, currently set as 20000 PyMC moves, which may take a while on travis. How about we augment the `argparser` on `quickmodel`'s so that we can specify far fewer moves on travis? Parsing the number of MCMC moves to `quickmodel` will make it easier to use as well.